src/Elmish.WPF.Tests/StaticViewModelTests.fs

```diff
@@ -0,0 +1,347 @@
+module StaticViewModelTests.M
+
+open System
+open System.Collections.Concurrent
+open System.Collections.ObjectModel
+open System.Collections.Specialized
+open System.ComponentModel
+open System.Windows.Input
+
+open Xunit
+open Hedgehog
+open Swensen.Unquote
+
+open Elmish.WPF
+
+
+
+type internal TestVm<'model, 'msg, 'B1>(model, binding: string -> Binding<'model,'msg>) as this =
+  inherit ViewModelBase<'model, 'msg>({ initialModel = model; dispatch = (fun x -> this.Dispatch x); loggingArgs = LoggingViewModelArgs.none })
+
+  let pcTriggers = ConcurrentDictionary<string, int>()
+  let ecTriggers = ConcurrentDictionary<string, int>()
+  let ccTriggers = ConcurrentDictionary<string, NotifyCollectionChangedEventArgs list>()
+  let cecTriggers = ConcurrentDictionary<string, int>()
+  let dispatchMsgs = ResizeArray<'msg> ()
+
+
+  do
+    (this :> INotifyPropertyChanged).PropertyChanged.Add (fun e ->
+      pcTriggers.AddOrUpdate(e.PropertyName, 1, (fun _ count -> count + 1)) |> ignore
+    )
+
+    (this :> INotifyDataErrorInfo).ErrorsChanged.Add (fun e ->
+      ecTriggers.AddOrUpdate(e.PropertyName, 1, (fun _ count -> count + 1)) |> ignore
+    )
+
+  member _.UpdateModel(m) = IViewModel.updateModel(this, m)
+
+  member x.GetPropertyName = nameof(x.GetProperty)
+  member _.GetProperty = base.Get<'B1>() binding
+
+  member private __.Dispatch x =
+    dispatchMsgs.Add x
+
+  member __.NumPcTriggersFor propName =
+    pcTriggers.TryGetValue propName |> snd
+
+  member __.NumEcTriggersFor propName =
+    ecTriggers.TryGetValue propName |> snd
+
+  member __.NumCcTriggersFor propName =
+    ccTriggers.GetOrAdd(propName, []).Length
+
+  member __.NumCecTriggersFor propName =
+    cecTriggers.TryGetValue propName |> snd
+
+  member __.Dispatches =
+    dispatchMsgs |> Seq.toList
+
+  member __.CcTriggersFor propName =
+    ccTriggers.TryGetValue propName |> snd |> Seq.toList
```

Comment:
```suggestion
  member private _.Dispatch x =
    dispatchMsgs.Add x

  member _.NumPcTriggersFor propName =
    pcTriggers.TryGetValue propName |> snd

  member _.NumEcTriggersFor propName =
    ecTriggers.TryGetValue propName |> snd

  member _.NumCcTriggersFor propName =
    ccTriggers.GetOrAdd(propName, []).Length

  member _.NumCecTriggersFor propName =
    cecTriggers.TryGetValue propName |> snd

  member _.Dispatches =
    dispatchMsgs |> Seq.toList

  member _.CcTriggersFor propName =
    ccTriggers.TryGetValue propName |> snd |> Seq.toList
```

Reply:
This is copied from the other tests, I'd rather not fix it separately.